### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by anssi55**
Modified category entity's description lenght to able to be longer.
Modified modify functions.
Post entity's datetime is now called created.
App now adds 4 premade categories if the database doesn't have any.
Readme updated according to changes.

### CODE REVIEW

File: app/dto/CreateCategory.ts
Comment (henrikra) [inline app/dto/CreateCategory.ts]: Maybe put this value somewhere as variable so you don't have to remember to update in `UpdateCategory.ts` also :D

File: 1566558511830-SeedCategory.ts
Comment (henrikra) [inline 1566558511830-SeedCategory.ts:3]: Is this file used? :D

**Review by henrikra — APPROVED** [review]
Looks good to me! Just fix these minor things and it is good to go! Remember to check after merge that production still works 💪

File: 1566558511830-SeedCategory.ts
Comment (anssi55) [reply 1566558511830-SeedCategory.ts:3]: Yes. You can't even change that awful name :D

**Comment by henrikra** [comment]
Looks Good 👍 Merge time

**Pull request merged**
